charts/castai-pod-node-lifecycle/values.yaml

```diff
@@ -54,6 +54,18 @@ telemetry:
   # Can be used for testing to skip the secret-api.yaml and use your local IDE.
   enabled: false
 
+metrics: 
+  # Can be used to disable the metrics function if there are conflicts
+  enabled: true
+
+hostNetwork: {}
+  # Enable host networking for mutating webhook
+  #enabled: true
```

Comment:
bunch of commented out things here and bellow

Reply:
Fixed in newest push.